internal/command/types.go

```diff
@@ -41,7 +42,8 @@ func (params *KameletPluginParams) Initialize() {
 }
 
 func (params *KameletPluginParams) newKameletClient() (camelkv1alpha1.CamelV1alpha1Interface, error) {
-	client, err := camelk.NewClient(true)
+	// TODO: This needs the Rest config plumbed through or alternatively look it up afresh.
+	client, err := camelk.NewForConfig(nil)
```

Comment:
This is essentially the "raw" client-go generated client. I'd hope that the kn plugin arch allows for plumbing through the rest config, if not you can look it up yourself.